connector/src/main/scala/com/datastax/spark/connector/util/ConfigCheck.scala

```diff
@@ -28,8 +27,7 @@ object ConfigCheck {
     CassandraSourceRelation,
     CassandraConnectorConf,
     AuthConfFactory,
-    CassandraConnectionFactory,
-    DseConnectionParams)
+    CassandraConnectionFactory)
```

Comment:
I'm not sure I understand the point of ConfObjects.  It doesn't appear to be used anywhere... is it just here to force realization of the various config parameter collections in order to achieve some kind of effect?

Reply:
The comment says
  /** Force creation of all Parameters. Any New Parameter Holding Objects should be added here **/

Which I honestly do not remember, maybe there was something about lazy initialization at some point in the past, where you wouldn't get certain parameters if the class was up yet ...

This codebase has gone through a  lot of changes, but I think it's to explicitly make sure we don't get a weird NPE below because we are trying to check properties and an object has init'd yet. Pretty sure that's it

Reply:
I was just wondering if this is something we could get rid of then, but I don't think it is.  The various param containers are using ConfigParameter.apply() to create their instances and that's what's used to populate the staticParameters collection.

:sadpanda:

Reply:
I'll delete it and see if everything still works :)